Projects/Features/SearchFeature/Sources/Reactors/SearchReactor.swift

```diff
@@ -26,8 +27,9 @@ final class SearchReactor: Reactor {
     }
 
     var initialState: State
-
-    init() {
+    private let service: any SearchCommonService
+    init(service: any SearchCommonService = DefaultSearchCommonService.shared) {
```

Comment:
```suggestion
    private let service: any SearchCommonService

    init(service: some SearchCommonService = DefaultSearchCommonService.shared) {
```
컴파일 타임에서 타입 확정 가능해서 some으로 둬도 되겠네요

Reply:
넵 반영하겠습니다.